SPECS/tpm2-tss/tpm2-tss.spec

```diff
@@ -1,51 +1,58 @@
-Summary:          OSS implementation of the TCG TPM2 Software Stack (TSS2)
-Name:             tpm2-tss
-Version:          2.4.0
-Release:          1%{?dist}
-License:          BSD
-URL:              https://github.com/tpm2-software/tpm2-tss
-Group:            System Environment/Security
-Vendor:           Microsoft Corporation
-Distribution:     Mariner
-Source0:          https://github.com/tpm2-software/tpm2-tss/releases/download/%{version}/%{name}-%{version}.tar.gz
-BuildRequires:    openssl-devel
-BuildRequires:    json-c-devel
-Requires:         openssl
-Requires:         json-c
-Requires(pre):    /usr/sbin/useradd /usr/sbin/groupadd
-Requires(postun): /usr/sbin/userdel /usr/sbin/groupdel
+Summary:        OSS implementation of the TCG TPM2 Software Stack (TSS2)
+Name:           tpm2-tss
+Version:        2.4.6
+Release:        1%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          System Environment/Security
+URL:            https://github.com/tpm2-software/tpm2-tss
+Source0:        https://github.com/tpm2-software/tpm2-tss/releases/download/%{version}/%{name}-%{version}.tar.gz
+
+BuildRequires:  json-c-devel
+BuildRequires:  openssl-devel
+
+Requires:       json-c
+Requires:       openssl
+Requires(postun): %{_sbindir}/groupdel
+Requires(postun): %{_sbindir}/userdel
+Requires(pre):  %{_sbindir}/groupadd
+Requires(pre):  %{_sbindir}/useradd
+
 %description
 OSS implementation of the TCG TPM2 Software Stack (TSS2)
 
 %package devel
-Summary:    The libraries and header files needed for TSS2 development.
-Requires:   %{name} = %{version}-%{release}
+Summary:        The libraries and header files needed for TSS2 development.
+Requires:       %{name} = %{version}-%{release}
+
 %description devel
 The libraries and header files needed for TSS2 development.
 
 %prep
-%setup -q
+%autosetup
+
 %build
 %configure \
     --disable-static \
     --disable-doxygen-doc \
-    --with-udevrulesdir=/etc/udev/rules.d
+    --with-udevrulesdir=%{_sysconfdir}/udev/rules.d
 
-make %{?_smp_mflags}
+%make_build
 
 %install
-make DESTDIR=%{buildroot} install
+%make_install
 
```

Comment:
do you think we can enable the check tests? 
https://github.com/tpm2-software/tpm2-tss#testing

Reply:
looks like the unit tests require cmocka which is not available yet. However, I see this being added in #1180

Reply:
I doubt I can get integration tests to work due to the dependency on `uthash`. I'll try to enable unit tests once we get a build with `cmocka` and its dependencies.